### PR TITLE
[docstring] Fix docstrings for `UniSpeechConfig`, `UniSpeechForCTC`, `UniSpeechSatConfig`, `UniSpeechSatForCTC` and `Wav2Vec2ForCTC`

### PR DESCRIPTION
**Pull request by gizemt**
Fixes #26638 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@ydshieh

### CODE REVIEW

File: src/transformers/models/wav2vec2/modeling_wav2vec2.py [hunk 1]
```diff
@@ -1871,6 +1871,11 @@ def forward(
 @add_start_docstrings(
     """Wav2Vec2 Model with a `language modeling` head on top for Connectionist Temporal Classification (CTC).""",
     WAV_2_VEC_2_START_DOCSTRING,
+    """
+        target_lang (`str`, *optional*):
+            Language id of adapter weights. Adapter weights are stored in the format adapter.<lang>.safetensors or
+            adapter.<lang>.bin If None, uses 'eng'.
```
Comment (ydshieh) [inline src/transformers/models/wav2vec2/modeling_wav2vec2.py:1877]: may I know why there is no `Only relevant when using an instance ...` part here?

Comment (gizemt) [reply src/transformers/models/wav2vec2/modeling_wav2vec2.py:1877]: not intentional. thanks for the catch!

**Comment by ydshieh** [comment]
Hi @gizemt I think the commit history is messed up when you merged the branch into this PR. You can see irrelevant changes to this PR shown up in `Files changed`.

Could you resolve this please?

**Comment by gizemt** [comment]
oops sorry, messed up rebase. fixed it now. @ydshieh

File: src/transformers/models/wav2vec2/modeling_wav2vec2.py [hunk 2]
```diff
@@ -1871,6 +1871,12 @@ def forward(
 @add_start_docstrings(
     """Wav2Vec2 Model with a `language modeling` head on top for Connectionist Temporal Classification (CTC).""",
     WAV_2_VEC_2_START_DOCSTRING,
+    """
+        target_lang (`str`, *optional*):
+            Language id of adapter weights. Adapter weights are stored in the format adapter.<lang>.safetensors or
+            adapter.<lang>.bin. Only relevant when using an instance of [`Wav2Vec2ForCTC`] or
+            [`Wav2Vec2PreTrainedModel`] with adapters. If None, uses 'eng'.
+    """,
```
Comment (ydshieh) [inline src/transformers/models/wav2vec2/modeling_wav2vec2.py:1879]:
```suggestion
    """
        target_lang (`str`, *optional*):
            Language id of adapter weights. Adapter weights are stored in the format adapter.<lang>.safetensors or
            adapter.<lang>.bin. Only relevant when using an instance of [`Wav2Vec2ForCTC`] with adapters. Uses 'eng'
            by default.
    """,
```

**Review by ydshieh — COMMENTED** [review]
A final tiny change to make and we are ready go go 🚀 .

**Comment by gizemt** [comment]
good to go @ydshieh thank you!

**Review by ydshieh — APPROVED** [review]
Thank you @gizemt for this contribution 🚀 !

**Pull request merged**